src/conways/conways_game.rs

```diff
@@ -0,0 +1,139 @@
+use std::collections::HashSet;
+
+use super::point::Point;
+
+pub struct ConwaysGame {
+    cells: HashSet<Point>,
+}
+
+impl ConwaysGame {
+    pub fn new(cells: Vec<Point>) -> Self {
```

Comment:
I like the `Point` entity, it makes the positioning clear. Now, I can't infer if when creating a `ConwaysGame` the cells are initialized as dead or alive, or if the vector of points is the vector of alive cells.

Reply:
Yes, you are right, i'm changing the variable to a more descriptive name